core/game_object.lua

```diff
@@ -102,16 +102,12 @@ function loadAPIs()
 
             -- Setup Localize text
             o:process_loc_text()
-
-            sendInfoMessage(
-                ('Registered game object %s of type %s')
-                :format(o.key, o.set), o.set or 'GameObject'
-            )
+            if not o.no_log then sendInfoMessage(('Registered game object %s of type %s'):format(o.key, o.set), o.set or 'GameObject') end
         end
     end
 
     --- Takes control of vanilla objects. Child class must implement get_obj for this to function.
-    function SMODS.GameObject:take_ownership(key, obj, silent)
+    function SMODS.GameObject:take_ownership(key, obj, silent, no_log)
```

Comment:
for the no-logging functionality, I was thinking of a setting on the logging system, not a setting per-object. basically, some toggle that temporarily prevents logging. It could be as simple as a global variable- that's not extensible, but it's also simple enough to be fine